### ISSUE TITLE
Vignette breaks down with lack of islandSeqs.txt page 3

### ISSUE THREAD

**Issue by methornton** [scharch/zap#6]
Hello!  Thank you for your wonderful program.  I am running through the vignette and I ran into a problem.  The generation of the islandSeqs.txt file from the processed data from SRR1057705 and SRR1057707 was not discussed.  It is necessary to proceed with the tutorial.  Also the sonar command 'getfasta' must be changed to getFastaFromList due to conflicts with other getfasta

```
sonar getfasta -l output/tables/islandSeqs.txt -f output/sequences/nucleotide/cap256-week34H_goodVJ_unique.fa -o output/sequences/nucleotide/cap256-week34H_islandSeq.fa
Input program 'getfasta' is unclear. Did you mean one of the following?
	getFastaFromAIRR.py
	getFastaFromList.py
	getReadsByAnnotation.py
	getListFromFasta.py
	2.2-get_island_interactive.R
root@a714d8cfed7e:/host/home/cap256-week34H# sonar getFastaFromList -l output/tables/islandSeqs.txt -f output/sequences/nucleotide/cap256-week34H_goodVJ_unique.fa -o output/sequences/nucleotide/cap256-week34H_islandSeq.fa

Mon Mar  9 20:49:01 2020 -- SONAR v4.2-29-g2c920bf-dirty run with command:
	/SONAR/utilities/getFastaFromList.py -l output/tables/islandSeqs.txt -f output/sequences/nucleotide/cap256-week34H_goodVJ_unique.fa -o output/sequences/nucleotide/cap256-week34H_islandSeq.fa

Traceback (most recent call last):
  File "/SONAR/utilities/getFastaFromList.py", line 92, in <module>
    main()
  File "/SONAR/utilities/getFastaFromList.py", line 67, in main
    for line in fileinput.input(arguments['-l']):
  File "/usr/lib/python3.6/fileinput.py", line 250, in __next__
    line = self._readline()
  File "/usr/lib/python3.6/fileinput.py", line 362, in _readline
    self._file = open(self._filename, self._mode)
FileNotFoundError: [Errno 2] No such file or directory: 'output/tables/islandSeqs.txt'
```
Could you provide the necessary commands to produce the islandSeqs.txt file?

**Comment by methornton** [comment]
I think I found it.  I am going through the '/SONAR/sample_data/runVignette.sh' script now.

**Comment by scharch** [comment]
Yeah, `islandSeqs.txt` is produced using the [interactive plotting script](https://github.com/scharch/SONAR/blob/master/lineage/2.2-get_island_interactive.R) to draw a box around the island, and so requires Docker to be run with `-e DISPLAY=$DISPLAY`. If not, `runVignette.sh` will instead use precomputed lineage members stored in the `sample_data` directory.

Thanks for catching the ambiguity of `getfasta`!

**Comment by methornton** [comment]
Hello!  I was having abit of trouble using the Docker image on a remote workstation due to display issues.  I use Ubuntu locally and so I installed the prerequisites from the Dockerfile.  I ran the tests script and everything passed.  I wanted to run the Vignette, mainly because that was what I was trying to do on the remote workstation and here is some things that I found that may help others. I needed to add SONAR=\path\to\SONAR\root\folder; export SONAR, to my .bashrc. Then I had to change '/SONAR/sample_data/...' to '${SONAR}/sample_data...' in the runVignette.sh script. I also needed to Add 'Sys.setenv("DISPLAY"=":0")' to my .Rprofile.  I have a question about how best to set the areas? 
![Area1_test](https://user-images.githubusercontent.com/5246174/76454296-a81b7180-6391-11ea-9478-5776194bc39a.png)
![Area1_2](https://user-images.githubusercontent.com/5246174/76454306-ae115280-6391-11ea-8eb5-c67b780b67cc.png)
![Area2_result](https://user-images.githubusercontent.com/5246174/76454316-b1a4d980-6391-11ea-8f57-b13d48e7cf6f.png)
Here are my images should they be tight to the higher numbers of transcripts.  Is the balck line a recommended value?

**Comment by scharch** [comment]
@methornton I see, yeah, I didn't really plan for someone to run the vignette outside of Docker. I will try to remember to add a bit of code for that case.

As far as actually running 2.2, you should be drawing a boundary around the "island" near 100% identity. You are instead capturing the unrelated bulk transcripts. There are some figures in the sample_data folder that show what it should look like.

**Comment by methornton** [comment]
Hello again!  Thank you!  I've fixed that and I've been all the way through the vignette.  There was a final issue.  The script makes plots directories, but for the last tree the  command sends the output to a 'plot' directory, "/usr/local/lib/sonar/SONAR/plotting/4.4-display_tree.py -t output/cap256-longitudinal_igphyml.flipped.tree -n /usr/local/lib/sonar/SONAR/sample_data/natives.csv -o output/plot/CAP256_SONAR-vignette_tree.png --sp 2 --sc 1500 -f 2".  I made a 'plot' directory and reran the line and it produced this very nice phylogenetic tree.
![CAP256_SONAR-vignette_tree](https://user-images.githubusercontent.com/5246174/76472952-51735f00-63b4-11ea-869a-763cc6fc9d9b.png)

Were the islandseq images supposed to be saved in the other cap256-week folders?
